main.go

```diff
@@ -18,79 +18,235 @@ package main
+								},
+								JsonPath: ".status.phase",
+								Pattern:  "Running",
+							},
+						},
+					},
+					{
+						Path: "https://storage.googleapis.com/knative-releases/serving/previous/v0.3.0/serving.yaml",
+						Name: "serving",
+						Namespace: "knative-serving",
+						Checks: []crd.ResourceChecks {
+							{
+								Kind:      "Pod",
+								Selector: metav1.LabelSelector{
+									MatchLabels: map[string]string{"app": "activator"},
+								},
+								JsonPath: ".status.phase",
+								Pattern:  "Running",
+							},
+							{
+								Kind:      "Pod",
+								Selector: metav1.LabelSelector{
+									MatchLabels: map[string]string{"app": "autoscaler"},
+								},
+								JsonPath: ".status.phase",
+								Pattern:  "Running",
+							},
+							{
+								Kind: "Pod",
+								Selector: metav1.LabelSelector{
+									MatchLabels: map[string]string{"app": "controller"},
+								},
+								JsonPath: ".status.phase",
+								Pattern:  "Running",
+							},
+							{
+								Kind: "Pod",
+								Selector: metav1.LabelSelector{
+									MatchLabels: map[string]string{"app": "webhook"},
+								},
+								JsonPath: ".status.phase",
+								Pattern:  "Running",
+							},
+						},
+					},
+					{
+						Path: "https://storage.googleapis.com/knative-releases/eventing/previous/v0.3.0/eventing.yaml",
+						Name: "eventing",
+						Namespace: "knative-eventing",
+						Checks: []crd.ResourceChecks {
+							{
+								Kind: "Pod",
+								Selector: metav1.LabelSelector{
+									MatchLabels: map[string]string{"app": "eventing-controller"},
+								},
+								JsonPath: ".status.phase",
+								Pattern:  "Running",
+							},
+							{
+								Kind: "Pod",
+								Selector: metav1.LabelSelector{
+									MatchLabels: map[string]string{"app": "webhook"},
+								},
+								JsonPath: ".status.phase",
+								Pattern:  "Running",
+							},
+						},
+					},
+					{
+						Path: "https://storage.googleapis.com/knative-releases/eventing/previous/v0.3.0/in-memory-channel.yaml",
+						Name: "eventing-in-memory-channel",
+						Namespace: "knative-eventing",
+						Checks: []crd.ResourceChecks{
+							{
+								Kind:      "Pod",
+								Selector: metav1.LabelSelector{
+									MatchLabels: map[string]string{"role": "dispatcher", "clusterChannelProvisioner": "in-memory-channel"},
+								},
+								JsonPath: ".status.phase",
+								Pattern:  "Running",
+							},
+							{
+								Kind: "Pod",
+								Selector: metav1.LabelSelector{
+									MatchLabels: map[string]string{"role": "controller", "clusterChannelProvisioner":"in-memory-channel"},
+								},
+								JsonPath: ".status.phase",
+								Pattern:  "Running",
+							},
+						},
+					},
+					{
+						Path: fmt.Sprintf("https://storage.googleapis.com/projectriff/riff-buildtemplate/riff-cnb-clusterbuildtemplate-%s.yaml", builderVersion),
+						Name: "riff-build-template",
+					},
+				},
+				Init: []crd.RiffResource {
```

Comment:
The current structure of the yaml is:
```
spec:
  resources: []
  init: []
```
I would like to change it to 
```
spec:
  resources:
    system: []
    initialization: []
```
I will create a new PR once this is merged.